Gemfile

```diff
@@ -173,9 +173,9 @@ gem "rails-timeago",           "2.11.0"
 
 gem "logging-rails", "0.5.0", require: "logging/rails"
 
-# Workarounds
-# https://github.com/rubyzip/rubyzip#important-note
-gem "zip-zip"
+# # Workarounds
+# # https://github.com/rubyzip/rubyzip#important-note
+gem "rubyzip"
```

Comment:
We lock down the version in your Gemfile to protect our users from too much impact when running `bundle update` on their own, please add `, "1.1.7"`